Chaincase.UI/ViewModels/SendViewModel.cs

```diff
@@ -111,15 +74,14 @@ public SendViewModel(Global global, SelectCoinsViewModel selectCoinsViewModel)
 
             Observable.FromEventPattern(SelectCoinsViewModel, nameof(SelectCoinsViewModel.SelectionChanged))
                 .ObserveOn(RxApp.MainThreadScheduler)
-                .Subscribe(_ => SetFees());
+                .Subscribe(_ => ApplyFees());
 
             _minMaxFeeTargetsEqual = this.WhenAnyValue(x => x.MinimumFeeTarget, x => x.MaximumFeeTarget, (x, y) => x == y)
                 .ToProperty(this, x => x.MinMaxFeeTargetsEqual, scheduler: RxApp.MainThreadScheduler);
 
             SetFeeTargetLimits();
             FeeTarget = Global.UiConfig.FeeTarget;
             FeeRate = new FeeRate((decimal)50); //50 sat/vByte placeholder til loads
-            SetFees();
```

Comment:
ApplyFees may not happen on first render because of this delete. Unless FeeTarget listener is sufficient

Reply:
in practice the AllFeeEstimate & fee target triggers ApplyFees()